codeexamples/SYCLUsingCUDAStaticLibraryOfKernels/StaticLibraryBuildCUDAKernels/bin/Makefile

```diff
@@ -0,0 +1,28 @@
+# Library names
+SRC_FILE ?= library.cu
+DST_FILE ?= libStaticLibOfFnsCUDAKernel
+
+# Location of the CUDA Toolkit
+CUDA_PATH ?= /usr/local/cuda/bin
+
+INCLUDE_PATH ?= <change this path to>/cuda-samples-nvidia/Common
+
+# Target rules
+all: build_debug
+
+build_debug:
+	$(CUDA_PATH)/nvcc -rdc=true -c -ccbin g++ -I$(INCLUDE_PATH) -I/usr/local/cuda/bin/../targets/x86_64-linux/include -m64 --threads 0 --std=c++11 -gencode arch=compute_75,code=sm_75 -g -G -O0 ../src/$(SRC_FILE) -o $(DST_FILE)_d_tmp.o 
+	$(CUDA_PATH)/nvcc -dlink -o  libStaticLibOfFnsCUDAKernel_d.o  libStaticLibOfFnsCUDAKernel_d_tmp.o -lcudart
+	ar cru  $(DST_FILE)_d.a  $(DST_FILE)_d.o $(DST_FILE)_d_tmp.o
+	ranlib $(DST_FILE)_d.a
+
+build:
+	$(CUDA_PATH)/nvcc -rdc=true -c -ccbin g++ -I$(INCLUDE_PATH) -I/usr/local/cuda/bin/../targets/x86_64-linux/include -m64 --std=c++11 -gencode arch=compute_75,code=sm_75 ../src/$(SRC_FILE) -o $(DST_FILE)_tmp.o 

```

Comment:
I don't think these hardcoded paths are always going to be correct, but even so we should at least normalise them

Reply:
These paths are defined by the CUDA toolkit's installation, not by me, and so should be common to everyone relative a to root install directory. What do you mean by 'normalise them'?

Reply:
In this context "normalise" means removing removing the relative components of the path, though I guess technically that might not actually be the same directory because of links. That being said this path is making a lot of assumptions anyway so it might be OK.